style.css

```
@@ -1,12 +1,11 @@
-body{
-    background-color: black;
-}
 h1{
-    background-color: white;
-}
-.odd{
-    color: green;
+    border :3px solid hsl(127, 84%, 51%)
```

Missin ; at the end of the line
```suggestion
    border: 3px solid hsl(127, 84%, 51%);
```